front/lib/api/assistant/agent.ts

```diff
@@ -390,11 +398,15 @@ async function* runMultiActionsAgent(
 
   const MIN_GENERATION_TOKENS = 2048;
 
-  const emulatedActions = await getEmulatedAgentMessageActions(auth, {
+  const { emulatedActions, jitActions } = await getEmulatedAndJITActions(auth, {
```

Comment:
If you do it here for JIT actions, you'll not list the jitActions in the prompt built in `runMultiActionsAgent()` (which is fine for emulated actions, less for the JIT actions imho).

Reply:
which prompt?

Reply:
Does not really use anything other than the presence of action but that's easy to move in between the injection?

Reply:
I still think it makes more sense to inject the real JIT actions as part of the agent configuration actions. They are just as real as others (such as the ones from global agent).

Why do you want to bundle them with the emulated ones (which can have some totally unrelated to JIT in the future).

Reply:
Main motivation is to minimize impact on agent.ts for now. Also the two really are related today :)